src/main/java/emissary/admin/Startup.java

```diff
@@ -382,8 +384,16 @@ protected boolean placeSetup(final int directoryActionArg, final Map<String, Str
                     if (p != null) {
                         placesArg.put(thePlaceLocation, thePlaceLocation);
                     } else {
-                        Startup.this.placesToStart.remove(thePlaceLocation);
-                        logger.debug("Giving up on {}", thePlaceLocation);
+
+                        if (Boolean.parseBoolean(System.getProperty("strict.mode"))) {
+                            logger.error("{} failed to start!", thePlaceLocation);
+                            Startup.this.failedPlaces.add(thePlaceLocation);
+                            Startup.this.placesToStart.remove(thePlaceLocation);
```

Comment:
I was a bit torn on this.  That's certainly an option that's easy to implement, but my thinking was if we get a complete list of the places that failed to start, it could help with potential debugging and prevent us from constantly building/running and watching different places crash.  I haven't been around long enough to know if that's a common issue though, so I can change it if need be.

Reply:
Yeah, would be good to get some additional opinions? I personally don't have strong feelings either way, because you're thought process is completely reasonable.  *May* want this to be configurable, but that would probably be a follow-on PR if needed

Reply:
I also don't have a strong opinion. I can see a use case for both ways. 
I support this being a follow on PR. In line with our goal small incremental changes :)

Reply:
I prefer fail fast but not strongly.